android/app/src/main/java/com/created/team201/presentation/createStudy/model/FragmentType.kt

```diff
@@ -0,0 +1,5 @@
+package com.created.team201.presentation.createStudy.model
+
+enum class FragmentType {
+    FIRST, SECOND;
+}
```

Comment:
```suggestion
enum class FragmentType {
    FIRST, SECOND
}
```

불필요한 세미콜론은 제거하는게 좋겠네요!

Reply:
궁금한건데 Enum Class에서 동반객체 쓸 때 상수에 세미콜론 붙인 뒤에 쓸 수 있어서 트레일링 콤마느낌인건가 라고 생각을 했는데 세미콜론이 필요 없을 때는 없는게 보편적인건가용?